### PR TITLE
Update a11y sections of both popover=hint and interest invokers

### PR DESCRIPTION
**Pull request by mfreed7**
This cleans up the "a11y" section of the `popover=hint` explainer by removing things related to "hover triggering". These are moved over to the interest invokers explainer, and a link is added there for reference.

### CODE REVIEW

**Comment by mfreed7** [comment]
+@aleventhal for review.

File: site/src/pages/components/interest-invokers.explainer.mdx [hunk 1]
```diff
@@ -189,6 +189,10 @@ for more.
...
+
+In particular, for interest-based triggering of popovers, there is a problem with hidden objects. In general, the details relation does not work for hidden objects, and we can’t necessarily expect the popover to be complete before it becomes visible, as that may happen on-demand. So what happens if the user navigates to a popover’s trigger, and it’s not open. If they tabbed there, then we can make sure the popover is shown on focus. But, if they are navigating with a virtual buffer as is likely, how will they trigger the hint popover to open (the user would need to know to focus the trigger, or JAWS/NVDA would need to start focusing objects during navigation as other screen readers do), and then and how will the user know it’s there (should we use a new value of the haspopup object attribute for these, e.g. `haspopup=hint`)?
```
Comment (aleventhal) [inline site/src/pages/components/interest-invokers.explainer.mdx:194]: Filed https://issues.chromium.org/issues/377923492

**Review by aleventhal — APPROVED** [review]

File: site/src/pages/components/interest-invokers.explainer.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (mfreed7) [reply site/src/pages/components/interest-invokers.explainer.mdx]: Done.

Comment (mfreed7) [reply site/src/pages/components/interest-invokers.explainer.mdx]: Thanks! I wasn't sure if you wanted this included, so I went ahead and included it.

**Review by mfreed7 — COMMENTED** [review]
Thanks!

**Comment by mfreed7** [comment]
@dbaron @lukewarlow @keithamus @gregwhitworth could someone please give me a rubber stamp to land this one?

**Review by keithamus — APPROVED** [review]

**Pull request merged**